src/components/ReportActionItemIOUPreview.js

```diff
@@ -41,26 +47,28 @@ const propTypes = {
         // This is either the user's full name, or their login if full name is an empty string
         displayName: PropTypes.string.isRequired,
     })).isRequired,
-
-    // Session info for the currently logged in user.
-    session: PropTypes.shape({
-        // Currently logged in user email
-        email: PropTypes.string,
-    }).isRequired,
 };
 
 const defaultProps = {
     iou: {},
+    iouReportID: undefined,
+    shouldHidePayButton: false,
+    onPayButtonPressed: null,
 };
 
 const ReportActionItemIOUPreview = ({
-    action,
-    isMostRecentIOUReportAction,
-    hasOutstandingIOU,
     iou,
     personalDetails,
     session,
+    shouldHidePayButton,
+    onPayButtonPressed,
 }) => {
+    const sessionEmail = lodashGet(session, 'email', null);
+
+    // Pay button should be visible to manager person in the report
+    // Check if the currently logged in user is the manager.
+    const isCurrentUserManager = iou.managerEmail === sessionEmail;
```

Comment:
NAB, this and the other details seem like something we could have potentially saved to the `iou` object in Onyx instead of doing it here. I think it's fine this way, but that might have been an interesting way to clean up the logic in the view.

Reply:
Ah, this is a good point -- I'm going to add this to the post-release refactor list as I don't see any reason to not make this improvement.